reference/content-types/smart_content.rst

```diff
@@ -303,65 +183,37 @@ Alias: "account"
 
 This provider filters the accounts.
 
-**Return Value**
+Media
+~~~~~
+
+Alias: "media"
+
+This provider filters the media.
+
+
+**Parameters**
 
 .. list-table::
     :header-rows: 1
 
-    * - resource
-      - Account
-      - The full entity from the database.
-    * - number
-      - string
-      - Property of the account.
-    * - name
-      - string
-      - Property of the account.
-    * - registerNumber
-      - string
-      - Property of the account.
-    * - placeOfJurisdiction
-      - string
-      - Property of the account.
-    * - uid
-      - string
-      - Property of the account.
-    * - corporation
-      - string
-      - Property of the account.
-    * - created
-      - string
-      - Property of the account.
-    * - creator
-      - string
-      - Property of the account.
-    * - changed
+    * - Parameter
+      - Type
+      - Description
+    * - mimetype_parameter
       - string
-      - Property of the account.
-    * - changer
+      - name of mime-type GET parameter (default: `mimetype`)
+    * - type_parameter
       - string
-      - Property of the account.
-    * - medias
-      - array
-      - Medias of the account.
-    * - emails
-      - array
-      - Property of the account.
-    * - phones
-      - array
-      - Property of the account.
-    * - faxes
-      - array
-      - Property of the account.
-    * - urls
-      - array
-      - Property of the account.
-    * - tags
-      - array
-      - Property of the account.
-    * - categories
-      - array
-      - Property of the account.
```

Comment:
this is very hard to update all the time i have added the hint that the developer should take a look in the dump of the data.